qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -14,3 +14,25 @@
     test_tags = "evict enlarge"
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
+    variants:
+        - @default:
+        - with_reboot:
```

Comment:
I think this part should be rearranged, `repeats` and `shell` repeat too many times